libs/core/algorithms/include/hpx/parallel/algorithms/partition.hpp

```diff
@@ -335,18 +335,15 @@ namespace hpx {
     /// calling thread.
     ///
     /// \returns  The \a partition_copy algorithm returns
-    ///           \a tagged_tuple<tag::in(InIter), tag::out1(OutIter1),
-    ///           tag::out2(OutIter2)>.
-    ///           The \a partition_copy algorithm returns the tuple of
-    ///           the source iterator \a last,
+    ///           \a std::pair<OutIter1, OutIter2>.
+    ///           The \a partition_copy algorithm returns the pair of
     ///           the destination iterator to the end of the \a dest_true
     ///           range, and the destination iterator to the end of the \a
     ///           dest_false range.
     ///
     template <typename FwdIter1, typename FwdIter2,
         typename FwdIter3, typename Pred, typename Proj>
-    hpx::util::tagged_tuple<tag::in(FwdIter1), tag::out1(FwdIter2),
-            tag::out2(FwdIter3)>
+    std::pair<FwdIter2 ,FwdIter3>
```

Comment:
Should we introduce a `partition_copy_result` type instead?

Reply:
... or `in_out_out`, analogous to what's already in place.

Reply:
The standard specifies std::pair for the non ranges overloads, the container ones do have in_out_out as the return type.